src/services/exile_service.py

```diff
@@ -142,18 +143,52 @@ async def get_user_exiles(logging_embed: discord.Embed, user: discord.User) -> s
 
     if len(exile_list) == 0:
         return "No exiles found for user"
+    else:
+        result = f"Exiles found for <@{user.id}>:"
+        for exile in exile_list:
+            exile_id = exile.exile_id
+            exile_reason = exile.reason
+            exile_start_epoch = round(exile.start_timestamp.timestamp())
+            exile_end_epoch = (
+                round(exile.end_timestamp.timestamp()) if exile.exile_status else None
+            )
+            exile_start_date = f"<t:{exile_start_epoch}:F>"
+            exile_end_date = (
+                f"<t:{exile_end_epoch}:F>" if exile_end_epoch else "Indefinite"
```

Comment:
don't think we're supporting indefinite anymore

Reply:
We might still have some records in the DB that are indefinite, I'm not sure. We should eventually go through and just make those be 1 year and remove the indefinite corner cases IMO

Reply:
https://naurffxiv.atlassian.net/browse/MOD-109